neptune-core/src/models/state/archival_state.rs

```diff
@@ -795,17 +779,16 @@ impl ArchivalState {
 
     /// Return parent of tip block. Returns `None` iff tip is genesis block.
     pub(crate) async fn get_tip_parent(&self) -> Option<Block> {
-        let tip_digest = self
+        let tip_index_value = self
```

Comment:
why this name `tip_index_value`? I prefer the existing shadowing, as it was, since that reduces the number of variables in scope. Small namespaces ftw.